guake/terminal.py

```diff
@@ -121,6 +121,10 @@ def __init__(self, guake):
 
         self.setup_drag_and_drop()
 
+        exclude = {"GDK_BACKEND"}
+        self.envv = [f"{i}={os.environ[i]}" for i in os.environ if i not in exclude]
```

Comment:
I believe we cannot do it at new terminal spawn.

User may add/append new environment variable after they spawn the terminal,

e.g.

```
#
# after spawn new terminal
#

$ export FOO=bar

# Spawn new termianl

$ echo $FOO  # This won't show anything, because we didn't spawn with this env var

$
```

Reply:
Setting environment variables in the new terminal still works upon testing. Unless you mean setting new environment variables in the parent and having them auto propagate down to already running children? In which case I don't think that was happening already anyway.

Reply:
At first glance I though export variable will be set in spawn terminal, but it seems I'm wrong.

Could I sync with you the test step?

```
1. in terminal type `export FOO=bar`
2. right click & `split |` to spawn terminal
3. echo $FOO will show nothing in spawn terminal
```

Reply:
This is what happens yeah. The bug is caused by environment variables from the place that guake is initially run from being propagated to all spawned terminals, and in particular one line in main.py at line 45 that sets GDK_BACKEND in the code. Terminal splitting shouldn't have anything to do with the bug.

Reply:
Got it, thanks for you explain